### PR TITLE
chore: adding automatic installer script and revamping docs

### PR DESCRIPTION
**Pull request by subnetmarco**
Requires https://github.com/Kong/kuma/pull/650

### CODE REVIEW

File: docs/docs/0.4.0/installation/centos.md [hunk 1]
```diff
@@ -2,145 +2,104 @@
...
+$ kumactl config control-planes add --name=XYZ --address=http://{address-to-kuma}:5681
 ```
+:::
+::::
+
+You will notice that Kuma automatically creates a [`Mesh`](../../policies/mesh) entity with name `default`.
+
+### 4. Quickstart
+
+Congratulations! You have successfully installed Kuma on Kubernetes. In order to start using Kuma, it's time to check out the  quickstart guide for Universal deployments.
```
Comment (devadvocado) [inline docs/docs/0.4.0/installation/centos.md:105]: @subnetmarco need to remove mention of kubernetes

File: docs/docs/0.4.0/installation/debian.md [hunk 2]
```diff
@@ -2,145 +2,104 @@
...
-```sh
-$ ./kumactl get dataplanes
-MESH      NAME        TAGS
-default   dp-echo-1   service=echo
-
-$ ./kumactl inspect dataplanes
-MESH      NAME        TAGS              STATUS   LAST CONNECTED AGO   LAST UPDATED AGO   TOTAL UPDATES   TOTAL ERRORS
-default   dp-echo-1   service=echo      Online   19s                  18s                2               0
-```
+Congratulations! You have successfully installed Kuma on Kubernetes. In order to start using Kuma, it's time to check out the  quickstart guide for Universal deployments.
```
Comment (devadvocado) [inline docs/docs/0.4.0/installation/debian.md:105]: @subnetmarco need to remove mention of kubernetes

File: docs/docs/0.4.0/installation/redhat.md [hunk 3]
```diff
@@ -2,145 +2,104 @@
...
+$ kumactl config control-planes add --name=XYZ --address=http://{address-to-kuma}:5681
 ```
+:::
+::::
+
+You will notice that Kuma automatically creates a [`Mesh`](../../policies/mesh) entity with name `default`.
+
+### 4. Quickstart
+
+Congratulations! You have successfully installed Kuma on Kubernetes. In order to start using Kuma, it's time to check out the  quickstart guide for Universal deployments.
```
Comment (devadvocado) [inline docs/docs/0.4.0/installation/redhat.md:105]: @subnetmarco need to remove mention of kubernetes

File: docs/docs/0.4.0/installation/macos.md [hunk 4]
```diff
@@ -2,145 +2,104 @@
...
+$ kumactl config control-planes add --name=XYZ --address=http://{address-to-kuma}:5681
 ```
+:::
+::::
+
+You will notice that Kuma automatically creates a [`Mesh`](../../policies/mesh) entity with name `default`.
+
+### 4. Quickstart
+
+Congratulations! You have successfully installed Kuma on Kubernetes. In order to start using Kuma, it's time to check out the  quickstart guide for Universal deployments.
```
Comment (devadvocado) [inline docs/docs/0.4.0/installation/macos.md:105]: @subnetmarco need to remove mention of kubernetes

File: docs/docs/0.4.0/installation/ubuntu.md [hunk 5]
```diff
@@ -2,145 +2,104 @@
...
+$ kumactl config control-planes add --name=XYZ --address=http://{address-to-kuma}:5681
 ```
+:::
+::::
+
+You will notice that Kuma automatically creates a [`Mesh`](../../policies/mesh) entity with name `default`.
+
+### 4. Quickstart
+
+Congratulations! You have successfully installed Kuma on Kubernetes. In order to start using Kuma, it's time to check out the  quickstart guide for Universal deployments.
```
Comment (devadvocado) [inline docs/docs/0.4.0/installation/ubuntu.md:105]: @subnetmarco need to remove mention of kubernetes

**Comment by subnetmarco** [comment]
@lobkovilya please add your instructions on `docs/docs/0.4.0/installation/openshift.md`

**Comment by subnetmarco** [comment]
(We need to add OpenShift to the sidebar + Add logo)

**Pull request merged**